### PR TITLE
method build and tested

### PR DESCRIPTION
**Pull request by Tw1stedRain**

### CODE REVIEW

File: src/test/java/multibracketvalidation/MultiBracketValidationTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import org.junit.Test;
+
+import static multibracketvalidation.MultiBracketValidation.multiBracketValidation;
+import static org.junit.Assert.*;
+
+public class MultiBracketValidationTest {
+
+    @Test
+    public void testMultiBrackets() {
```
Comment (JessLovell) [inline src/test/java/multibracketvalidation/MultiBracketValidationTest.java:11]: Your test cases do not account for special cases. The instructions give some good edge cases to consider.

File: src/main/java/multibracketvalidation/MultiBracketValidation.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package multibracketvalidation;
+
+public class MultiBracketValidation {
+
+    public static boolean multiBracketValidation(String input) {
+        String br1 = "(";
```
Comment (JessLovell) [inline src/main/java/multibracketvalidation/MultiBracketValidation.java:6]: This code will fail when there are more than 1 set of matching brackets.

File: src/main/java/multibracketvalidation/MultiBracketValidation.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+package multibracketvalidation;
+
+public class MultiBracketValidation {
```
Comment (JessLovell) [inline src/main/java/multibracketvalidation/MultiBracketValidation.java:3]: The optimized solution for this is to use a stack and push open brackets onto the stack and pop when you come across a closing bracket.
